data/changelog.txt

```diff
@@ -1,30 +1,115 @@
 # 0.H
 
 ## Features:
+Prevent occlusions via (auto)-semi-transparent sprites (similar to retraction)
+flip overmap tiles to default true
```

Comment:
This'll probably end up in a 0.G point release, not sure it should be in the changelog because of that.
```suggestion
```